vendor/grpc-1_26_0/build.gradle

```diff
@@ -20,7 +20,7 @@ import org.apache.beam.gradle.GrpcVendoring
 
 plugins { id 'org.apache.beam.vendor-java' }
 
-description = "Apache Beam :: Vendored Dependencies :: gRPC :: 1.21.0"
+description = "Apache Beam :: Vendored Dependencies :: gRPC :: 1.26.0"
```

Comment:
I suggest adding source code comment “When upgrading the version, refer to http://...”